lighthouse-core/gather/drivers/driver.js

```diff
@@ -447,6 +447,59 @@ class Driver {
       storageTypes: 'all',
     });
   }
+
+  /**
+   * Tracks function call usage. Used by captureJSCalls to inject code into the page.
+   * @param {Function} func The function call to track.
+   * @param {Set} set An empty set to populate with stack traces. Should be
+   *     on the global object.
+   * @return {Function} A wrapper around the original function.
+   */
+  static captureUsage(func, set) {
```

Comment:
I don't think this should be in the `Driver` class since it's not a method for it. One option would be to define it outside of the class body to make that clear?